src/cli.js

```diff
@@ -236,6 +236,17 @@ Options
                 reporter.isReportServer = boolValue();
                 break;
 
+            case "report-root":
+                if(!value.match((/^(.*?)[\\\/]([^\\\/]+[a-zA-Z])$/) || (/^[^\/\: ]+(\:[0-9]+)?$/))) {
+                    utils.error(`Invalid path report folder. It must be absolute 'path', 'domain' or 'domain:port'.`);
+                }
+                reporter.reportPath = value;
+                break;
+
+            case "history":
+                reporter.history = boolValue();
+                break;
```

Comment:
You should also add descriptions of `report-root` and `history` into the big switch statement for `smashtest --help`

Reply:
Maybe rename `report-root` to something like `report-path`?